tests/conftest.py

```
@@ -1284,11 +1284,20 @@ def factory(
+            delete_storageclass_and_deregister,
+        )
+
         for instance in instances:
-            instance.delete()
-            instance.ocp.wait_for_delete(instance.name, timeout=120)
+            try:
+                delete_storageclass_and_deregister(
+                    sc_name=instance.name,
+                    sc_ocp=instance.ocp,
+                )
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)